tests/playbooks/tests_wireless_wpa3_owe.yml

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: wlan0
+  tasks:
+    - name: "INIT: wireless tests"
+      debug:
+        msg: "##################################################"
+    - include_tasks: tasks/setup_mock_wifi_wpa3_owe.yml
+    - block:
+        - name: "TEST: wireless connection with WPA3 Personal"
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_allow_restart: true
+            network_connections:
+              - name: "{{ interface }}"
+                state: up
+                type: wireless
+                ip:
+                  address:
+                    - 203.0.113.2/24
+                  dhcp4: "no"
+                  auto6: "no"
+                wireless:
+                  ssid: "hostapd-owe"
+                  key_mgmt: "owe"
+                  password: "12345678"
+    - name: Verify wireless profile KEY_MGMT entry
+      shell: |
+        set -euxo pipefail
+        cat /etc/sysconfig/network-scripts/ifcfg-wlan0 | grep KEY_MGMT
```

Comment:
This is incorrect on Fedora 33+ any more as they don't use ifcfg-plugin there.

Please use something like:

`nmcli -g ethtool.coalesce-rx-frames c show {{ interface }}`

Reply:
`nmcli c show {{ interface }} | grep 802-11-wireless-security.key-mgmt` works for me